src/components/liveblog/body.tsx

```diff
@@ -22,32 +25,43 @@ const LiveBodyStyles = (pillarStyles: PillarStyles): SerializedStyles => css`
 interface LiveblogBodyProps {
     pillar: Pillar;
     blocks: LiveBlock[];
-    imageSalt: string;
+    totalBodyBlocks: number;
+    imageMappings: ImageMappings;
 }
 
-const LiveblogBody = ({ pillar, blocks, imageSalt }: LiveblogBodyProps): JSX.Element => {
+const LiveblogBody = (props: LiveblogBodyProps): JSX.Element => {
+    const { pillar, blocks: initialBlocks, imageMappings, totalBodyBlocks } = props;
+    const [blocks] = useState(initialBlocks);
 
-    const initialBlocks = blocks.slice(0, 7);
-    const LoadMore = ({ total }: { total: number }): JSX.Element | null => total > 10
-        ? <LiveblogLoadMore pillar={pillar}/> 
+    async function loadMoreBlocks(): Promise<void> {
```

Comment:
Does this need to live within `LiveblogBody`? Can it exist at the top level of this module?

Reply:
The state is currently scoped to `liveblogBody` and that's the only component rehydrated so far so I'm not sure.

Eventually we'd want to do `setBlocks(...blocks, newBlocks)` after we fetch them.

Reply:
Oh but I can pass a function down, will try that.